src/Microsoft.AspNetCore.Mvc.Razor.Host/ViewComponentTagHelperPass.cs

```diff
@@ -11,21 +11,21 @@
 
 namespace Microsoft.AspNetCore.Mvc.Razor.Host
 {
-    public class ViewComponentTagHelperPass : IRazorIRPass
+    public class ViewComponentTagHelperPass : IRazorIROptimizationPass
     {
         public RazorEngine Engine { get; set; }
 
-        public int Order => RazorIRPass.LoweringOrder;
+        public int Order => RazorIRPass.DefaultFeatureOrder;
```

Comment:
The name is bad. We should probably just delete this. We shouldn't be referencing it anywhere in MVC.